src/main/java/com/yahoo/bullet/bql/tree/NodeLocation.java

```diff
@@ -10,37 +10,17 @@
  */
 package com.yahoo.bullet.bql.tree;
 
+import lombok.RequiredArgsConstructor;
+
+import static java.lang.String.format;
+
+@RequiredArgsConstructor
 public final class NodeLocation {
```

Comment:
Is it possible for this to have the whole BQL so you can use the character position to get some chars around the error to show some context in the error?

Reply:
what if we just built a feature into the ui that highlights the location in the original query string.. somehow

Reply:
Yea either way, this can be for another PR